code/game/objects/items/kitchen.dm

```diff
@@ -96,6 +96,23 @@
 	wound_bonus = -5
 	bare_wound_bonus = 10
 	tool_behaviour = TOOL_KNIFE
+	var/armor_pene_stab = 20
+	var/force_stabby = 5
+	var/wound_bonus_stabby = 5
+
+/obj/item/kitchen/knife/attack_alt(mob/living/victim, mob/living/user, params)
```

Comment:
I love the direction we're going in combat with right-click, good PR.
I'd say give stabbing a different sound - auditory cues in combat are great.